data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_base_qa_report1.py

```diff
@@ -54,14 +54,71 @@
 #
 #
 
-# ## step1
+# ## Step1
 # - Verify the following columns in the deid_cdr Observation table have been set to null:
-# o   value_as_string
-# o   value_source_value
+#   - value_as_string
+#   - value_source_value
 #
 # has been done in first sql for deid, can be skipped here
 
-# ## Query 1.2 Find person_ids in pre_dedi_com_cdr person table who have ethnicity_source_concept_id values AS 1586147  & race_source_concept_id AS ( 1586146 OR 1586142 OR 1586143) , then verify that the output in the deid_base_cdr observation table for that person_id after mapping  will results in 2-rows .
+# ## Query 1.1
+#
+# [DC-3597](https://precisionmedicineinitiative.atlassian.net/browse/DC-3597): Check if insurance selections are “Indian Health Services” (or a variant).
+#
+# Even though this option does not overtly identify a participant as AI/AN, it suggests so.  Therefore, these responses are generalized to "Other" at the registered tier de-id stage.
+#
+# The observation_concept_id's are:
+# - `40766241`
+#
+# - `1585389`
+#
+# - `43528428`
+
+# +
+query = JINJA_ENV.from_string("""
+SELECT observation_id FROM `{{project_id}}.{{deid_base_cdr}}.observation`
```

Comment:
@MichaelSchmidt82 this check looks for if there are any duplicates caused by the generalization which is good, but there should also be a check which finds out if we can find old(i.e Indian Health related) responses in the tables as well. I suggest you add another check which checks if the Indian health related(i.e old) value_source_concept_id and value_as_concept_id still exists in the table, if exists the check should fail and if you get no result the check should pass.